ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -311,7 +499,7 @@ export const RuntimePanel = fp.flow(withCurrentWorkspace(), withCdrVersions())((
       <FlexColumn style={{marginTop: '1rem'}}>
         <label htmlFor='runtime-compute'>Compute type</label>
         <Dropdown id='runtime-compute'
-                  disabled={!hasMicroarrayData}
+                  // disabled={!hasMicroarrayData}
```

Comment:
Just noticed this diff..

Reply:
fml i keep forgetting to revert this. will open new PR off master